raiden/tasks.py

```diff
@@ -1,17 +1,52 @@
 import time
-import structlog
+import requests
+from pkg_resources import parse_version
 
+import click
 import gevent
 from gevent.event import AsyncResult
-from gevent.queue import (
-    Queue,
-)
+from gevent.queue import Queue
+import structlog
+
 from raiden.exceptions import RaidenShuttingDown
+from raiden.utils import get_system_spec
+
+CHECK_VERSION_INTERVAL = 3 * 60 * 60
+LATEST = 'https://api.github.com/repos/raiden-network/raiden/releases/latest'
+RELEASE_PAGE = 'https://github.com/raiden-network/raiden/releases'
 
 REMOVE_CALLBACK = object()
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def check_version():
+    """Check every 3h for a new release"""
+    app_version = parse_version(get_system_spec()['raiden'])
+    while True:
+        try:
+            content = requests.get(LATEST).json()
+            # getting the latest release version
+            latest_release = parse_version(content['tag_name'])
+            # comparing it to the user's application
+            if app_version < latest_release:
+                click.secho('You\'re running version {}. The latest version is {}'
+                            .format(app_version,
+                                    latest_release,
+                                    ),
+                            fg='red',
+                            )
+                click.secho('It\'s time to update! Releases: {}'.format(RELEASE_PAGE), fg='red')
```

Comment:
Please re-format this block according to our style guide: 
- Break long statements after the opening paren
- Use double quotes if the string contains a single quote in the text